### DISCUSSION TITLE
File specified via "ios.googleServicesFile" field in your app.json is not checked in to your repository and won't be uploaded to the builder

### DISCUSSION THREAD

**Discussion by luklapp** [expo/expo#15958] [Archived: Questions]
Hi! I just started a build yesterday with `eas build`, which told me:

> File specified via "ios.googleServicesFile" field in your app.json is not checked in to your repository and won't be uploaded to the builder. 
> Use EAS Secret to pass all values that you don't want to include in your version control. Learn more.
> If you are using that file for compatibility with the classic build service (expo build) you can silence this warning by setting your build profile's env.GOOGLE_SERVICES_FILE in eas.json to any non-empty string.

But how can I upload a file to EAS secrets? The `ios.googleServicesFile` configuration expects a **path**, not the file content itself, so I probably can't put the file content into an environment variable? 

What is the correct way to use `eas build` with `ios.googleServicesFile`?

**Comment by bordot** [comment]
@luklapp 
I'm not sure if you found a solution for this, but I've found that modifying eas.json to include this did the trick:

```
"production": {
        "ios": {
          "env": {
            "GOOGLE_SERVICES_FILE": "./AppleService.plist"
          }
        }
    }


```

**Reply by batazo** [reply]
I tried this and it just crashes the whole app
